examples/using-typescript/src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Do we need this test? People who used cra will know this is a "left over". I would remove it.

Reply:
I think it could be beneficial to have at least one test that runs something on typescript so type errors can come up if we ever decide to change the typescript definition.

Reply:
Don't always come up during build time? (So building the example should be enough for this?)